packages/components/src/Popover/Popover.mdx

```diff
@@ -0,0 +1,195 @@
+---
+name: Popover
+menu: Components
+route: /components/popover
+showDirectoryLink: true
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Content } from "../Content";
+import { Heading } from "../Heading";
+import { Button } from "../Button";
+import { Checkbox } from "../Checkbox";
+import { useState, useRef } from "react";
+import { Popover } from ".";
+
+# Popover
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+Write a description about what Popover component is trying to achieve.
+
+```ts
+import { Popover } from "@jobber/components/Popover";
+```
+
+<Playground>
+  {() => {
+    const divRef = useRef();
+    const [showPopover, setShowPopover] = useState(false);
+    return (
+      <>
+        <span ref={divRef}>
+          <Button
+            label="Toggle Popover"
+            onClick={() => setShowPopover(!showPopover)}
+          />
+        </span>
+        <Popover
+          attachTo={divRef}
+          open={showPopover}
+          dismissible={true}
+          preferredPlacement="right"
+          onRequestClose={() => setShowPopover(false)}
+        >
+          <Content>Here is your first popover</Content>
+        </Popover>
+      </>
+    );
+  }}
+</Playground>
+
+<!--
+  It is not necessary to include Design & Usage Guidlelines. Use this section
+  to answer common questions and considerations about component usage.
+-->
+
+## Design & Usage Guidelines
+
+When contributing to, or consuming the Popover component, consider the
+following:
+
+### Usage
+
+- A popover should be used to introduce a novel UI element or highlight a change
+  to an existing interface.
+- A popover should contain information that is helpful, but not necessary for
+  the user to successfully interact with the page.
+- Once a popover has been shown, it should not show up again once the user has
+  interacted with it.
+- **You must use a `useRef` hook to create a reference point for the popover to
```

Comment:
I would consider this a technical note - it's getting buried here under the design guidelines. Can we explain this on the `attachTo` prop itself, where it's more relevant?

Reply:
sounds good!